README.md

```diff
@@ -20,51 +20,125 @@ custom_components/hasp-lvgl/const.py
+### Configuration Variables
+
+**hasp_lvgl:**\
+  *(Required)* Your platform identifier. Can be replaced with `hasp_lvgl: !include zz_etc/hasp_lvgl.yaml` which will allow you to store all further options in a separate configuration file located at `zz_etc/hasp_lvgl.yaml`.
+
+**plate_my_room:**\
+  *(Required)* Your plate identifier. For each plate in your sytem, such an entry is required, has to be unique.
+
+**topic:**\
+  *(string)* *(Required)* The MQTT topic your plate is configured with.
+
+**pages:**\
+  *(Optional)* Page navigation objects: `prev_obj`, `home_obj`, `next_obj` are the dedicated objects on the screen which will navigate the pages in previous, home and next directions, respectively. (_Note:_ objects on page `0`, have `p0` in their name, they appear on all pages).
+
+**objects:**\
+  *(Optional)* Definition of the objects reacting to changes in Home Assistant, or generating events for Home Assistant.
+
+**obj:**\
+   *(string)* *(Required)* The object identifier which we want to integrate with Home Assistant. Its name has the form `pXbY` where `X` represents the page where the object is located, and `Y` represents the `id` of the object on that page.
+
+**properties:**\
+  *(Optional)* List containing the properties of the object which we want to modify based on changes occurring in Home Assistant. In the example above `text` property gets updated whenever `sensor.my_room_temperature` changes. Various properties are available for the objects, full details in [hasp-lvgl documentation](https://fvanroie.github.io/hasp-docs/#objects/).
+  
+**event:**\
+  *(Optional)* List containing the events generated by the object when touched on the screen. These are object-specific, and can be observed accurately with an MQTT client. In the example above, when object `p1b2` (which is a toggle button) generates the `on` event, `light.my_room` will be turned on by the service call `light.turn_on` as specified in the event config. And similarily when `off` event comes through MQTT, the light will be turned off by the corresponding service call.
+
+_Note:_ Any variable coming from the MQTT message can be used between curly brackets, and passed to the service call. In the example above when object `p2b2` (which is a dropdown selector) generates the `changed` event, a persistent notification will appear in Home Assistant's Lovelace interface containing the selected text from the object, which was passed over from the MQTT message.
+
+### Component-specific services
+
+This component implements some specific services to make interactions with the plate even more comfortable.
+
+**hasp_lvgl.load_pages**\
+  Clears plate and loads new layout from pages.jsonl file at optional _full path_ (e.g. `/config/pages.jsonl` in casebof hassio). The file must be located in an authorised location deffined by [allowlist_external_dirs](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_dirs).
```

Comment:
```suggestion
  Clears plate and loads new layout from pages.jsonl file at optional _full path_ (e.g. `/config/pages.jsonl` in case of hassio). The file must be located in an authorised location defined by [allowlist_external_dirs](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_dirs).
```